process/txcost/gasUsed.go

```diff
@@ -6,6 +6,8 @@ import (
 	"github.com/multiversx/mx-chain-proxy-go/data"
 )
 
+const extraGasPercent = 0.15 // 0.15 means 15% more gas to add
```

Comment:
This will alter all results that came through `computeResponsesGasUsed` function. So the function will return misleading results. I suggest refactoring this solution.

Reply:
removed this part